classes/plugininfo/cleaner.php

```diff
@@ -148,6 +148,10 @@ public function get_settings_section_url() {
             return new \moodle_url('/local/datacleaner/cleaner/environment_matrix/index.php');
         }
 
+        if ($this->name == 'scheduled_tasks') {
+            return new \moodle_url('/local/datacleaner/cleaner/scheduled_tasks/index.php');
```

Comment:
I know there is a precedent with the environment_matrix (which should also be fixed), but this should be overridden in the plugin, not in the base class. The base should never be aware of the different cleaners